adapters/unicorn/unicorn.go

```diff
@@ -98,6 +107,15 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 	return []*adapters.RequestData{requestData}, nil
 }
 
+func isOtherThanJPYSpecified(cur []string) bool {
```

Comment:
Nitpick: But could this function name be renamed to something like `isCurOtherThanJPY` or `isOtherThanJPY`. I just think the function name could be a little more simple.